### PR TITLE
Rename

### PR DESCRIPTION
**Pull request by andrewdmcleod**

### CODE REVIEW

File: src/tests/bundles/bionic-queens-swift-v2.yaml [hunk 1]
```diff
@@ -33,7 +33,11 @@ relations:
 - - cinder-ceph:storage-backend
   - cinder:storage-backend
 - - cinder
+<<<<<<< HEAD
+  - cinder-backup-swift-proxy
+=======
   - cinder-backup-swift
+>>>>>>> master
```
Comment (fnordahl) [inline src/tests/bundles/bionic-queens-swift-v2.yaml]: There appears to be one more conflict to resolve

**Review by fnordahl — COMMENTED** [review]
See in-line comment about merge conflict traces

**Comment by fnordahl** [comment]
For reference, I have also proposed to carry the spec edited for the rename forward as approved for Ussuri: https://review.opendev.org/#/c/696087/

File: src/tests/bundles/bionic-queens-swift-v2.yaml (hunk 1 above)
Comment (andrewdmcleod) [reply src/tests/bundles/bionic-queens-swift-v2.yaml]: Fixed

**Pull request merged**